docs/accessibility/accessibility-module-feature-overview.md

```diff
@@ -4,18 +4,23 @@ Accessibility module feature overview
 The Accessibility module adds several layers of functionality for making your charts compatible with assistive technologies, improving the usability of your charts, and helping you reach a broader audience. 
 
 Keyboard navigation
--------------------
-
+--------------
 An essential part of the Accessibility module is the added keyboard navigation support. This is obviously a critical feature for users with mobility impairments, but also benefits all users. While using a computer mouse can be faster, keyboard navigation allows all users access to precise navigation of the chart.
 
-By including the Accessibility module with your charts, users are able to navigate and interact with the data points, chart menu, and other chart controls using the keyboard only.
+By including the Accessibility module with your charts, users are able to navigate and interact with the data points, chart menu, and other chart controls using the keyboard only. 
```

Comment:
> users are able

* Including the A... module allows users to...

> using the keyboard only

Sounds like mouse does not work.

* by using exclusively their keyboard.
* by using solely their keyboard.